cli/run_util.go

```diff
@@ -1027,8 +1027,8 @@ func runWorkflowWithConfiguration(
 		log.Warnf("Failed to trigger WillStartRun, error: %s", err)
 	}
 
-	//
 	buildRunResults := models.BuildRunResultsModel{
+		WorkflowID:     workflowToRunID,
```

Comment:
Mostly we print the workflow title. Shouldn't we work with that propeorty in this case too?

Reply:
Can we define `title` on the workflow too, not just steps? That's new to me 😅 (I was aware only of the `description` field).
In my use case I use the workflow ID anyway in the `bitrise run ...` command, so I'm interested in the ID specifically when looking at the logs.
Also, we have limited space here (80 chars), and I think even if a title is defined, it's typically longer than the ID.

Reply:
Alright, I'm fine with it.
Currently, we mix usage of the title and id for logs anyway.

(e.g.: [print running wf uses the title](https://github.com/bitrise-io/bitrise/blob/92bff8c14aac9b9e0cdba4c463bace3d9188521f/cli/run_util.go#L877), [print workflow chain uses the id](https://github.com/bitrise-io/bitrise/blob/92bff8c14aac9b9e0cdba4c463bace3d9188521f/cli/run.go#L155) )

It would be great to unify which propoerty we use for logging at some point.